### PR TITLE
Update Bootstrap logo, detection and implies

### PR DESCRIPTION
**Pull request by wxh06**
SVG logo clipped from the source code of <https://getbootstrap.com/#content>.

### CODE REVIEW

**Comment by wxh06** [comment]
@AliasIO BTW, is it necessary to add **[Popper](https://popper.js.org/)** detection? It's required by Bootstrap: <https://getbootstrap.com/docs/4.5/getting-started/introduction/#js>
According to its offical website, it's commonly used by companies such as **Microsoft**, **Atlassian**, **GitLab**, **Medium**, **Adobe**, **Stack Exchange**, **Drupal** and **Uber**.

File: src/apps.json [hunk 1]
```diff
@@ -1699,19 +1699,18 @@
       "cpe": "cpe:/a:getbootstrap:bootstrap",
       "html": [
         "<style>/\\*!\\* Bootstrap v(\\d\\.\\d\\.\\d)\\;version:\\1",
```
[the remark this answers is not in the document]
Comment (wxh06) [reply src/apps.json:1701]: This regex should match
```html
<style>
/*!
 * Bootstrap v4.5.0 (https://getbootstrap.com/)
 * Copyright 2011-2020 The Bootstrap Authors
 * Copyright 2011-2020 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 */
</style>
```
https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css

[the remark this answers is not in the document]
Comment (AliasIO) [reply src/apps.json:1701]: No, line breaks can't be matched. There is no good way to use a commented out section like that.

[the remark this answers is not in the document]
Comment (wxh06) [reply src/apps.json:1701]: I found a good way tom match whitespace: `\s`
```suggestion
        "<style>\\s+/\\*!\\s+\\* Bootstrap v(\\d\\.\\d\\.\\d)\\;version:\\1",
```

**Comment by wxh06** [comment]
@AliasIO It can be merged now

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**